cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutorTest.java

```diff
@@ -118,7 +118,8 @@ public void testReplicaReassignment() throws InterruptedException, OngoingExecut
     KafkaZkClient kafkaZkClient = KafkaCruiseControlUtils.createKafkaZkClient(zookeeper().connectionString(),
                                                                               "ExecutorTestMetricGroup",
                                                                               "ReplicaReassignment",
-                                                                              false);
+                                                                              false,
+                                                                              null);
```

Comment:
1. [Applies to the updates in `ReplicationThrottleHelperTest`, as well] In the main code, there is no code path that passes `null` to the `KafkaCruiseControlUtils#createKafkaZkClient`. Should we pass the return value of `ZKConfigUtils#zkClientConfigFromKafkaConfig` for some config to make the test more in line with the typical usage?
2. Would it be possible to add some tests to see that the new functionality works as expected (`CCKafkaIntegrationTestHarness` might be useful)?